scarlet/initialization.py

```diff
@@ -169,29 +169,19 @@ def trim_morphology(center_index, morph, bg_thresh):
     bbox = Box.from_bounds((bottom, top), (left, right))
     return morph, bbox
 
-
-def init_extended_source(
-    sky_coord,
+def morph_init(sky_coord,
     frame,
     observations,
+    seds,
     coadd=None,
     coadd_rms=None,
-    thresh=1,
-    symmetric=True,
+    symmetric=False,
```

Comment:
Don't simply change defaults for the initialization. `ExtendedSource` works well with it. If you don't like that, specify `symmetric=False` when you call this function.

Reply:
Yup, that was a mistake.I mixed up sources.